### DISCUSSION TITLE
nerdctl can not  load  tar.gz  images files, but docker does.

### DISCUSSION THREAD

**Discussion by tanguofu** [containerd/nerdctl#841] [Q&A] [answered]
[root@VM-74-16-centos artifacts]# nerdctl load -i 0972d84-20220222-image.tar.gz                                                                                                                                                    
FATA[0000] archive/tar: invalid tar header              
[root@VM-74-16-centos artifacts]# docker load -i 0972d84-20220222-image.tar.gz 
Loaded image: ****-0972d84-20220222

**Comment by AkihiroSuda** [comment, marked as answer]
Implemented in nerdctl v0.15.0.
Please upgrade your nerdctl.

https://github.com/containerd/nerdctl/commit/cdc3299f32cdcb17fc2e80ef2dc00cfdf2bbb169
